README.md

```diff
@@ -18,6 +18,9 @@ SESSION_TIMEOUT_SECONDS=43200
 SESSION_NAME_CLAIMS=userid
 SESSION_TAG_CLAIMS=sub,userid,user_name
 REDIRECT_URIS=https://.....,https://....
+MARKETPLACE_PRODUCT_CODE=xxxxxx
```

Comment:
Good suggestion: It's not yet clear to me how this approach will generalize to multiple products but doing this might help avoid "painting ourselves into a corner."